content/reference/user/artifact_gen/index.md

```diff
@@ -125,16 +127,16 @@ You can also exclude dependencies you don't want to ship and make the artifact l
 
 ### SBT Project
 
-In a sbt project, you have to add:
+In your SBT project configuration, you must add:
 
 - the Gatling dependencies
-- the sbt plugin for FrontLine, so that it can package your code into a deployable artifact
+- the Gatling Enterprise SBT plugin (`"io.gatling.frontline" % "sbt-frontline"`), used to package your code into a deployable artifact
```

Comment:
```suggestion
- the Gatling Enterprise SBT plugin (`"io.gatling.frontline" % "sbt-frontline"`), packages your code into a deployable artifact
```

Reply:
Not grammatically correct in this context. But it could be "[the plugin], **which** packages your code..."; that could work. 🤔

Reply:
I prefer ", which packages"